vision/benchmarks/accuracy/bench_module.py

```diff
@@ -165,7 +165,7 @@ def accuracy_get_module_roll(self, region: np.ndarray) -> float:
         return get_module_roll(region)
 
 
-def bench_module_accuracy(folder: str) -> None:
+def bench_module_accuracy(folder: str, quiet_output=True, draw_centers=False) -> None:
```

Comment:
quiet_output should probably default to False, since otherwise you would need to specify that you want it "loud" everytime, which is the reverse of what --quiet implies.

Reply:
Ah whoops, I meant to set that to false

Reply:
I switched the name and the bool value smh